docs/installation.md

```diff
@@ -227,10 +227,9 @@ datadog-agent-lkfqt                          0/1     Running    0          15s
 datadog-agent-zvdbw                          1/1     Running    0          8m1s
 ```
 
-## Install the `kubectl` plugin
-
-See the [`kubectl` plugin doc](/docs/kubectl-plugin.md)
+## Install the kubectl plugin
 
+See the [`kubectl` plugin documentation](/docs/kubectl-plugin.md).
```

Comment:
```suggestion
See the [`kubectl` plugin documentation](/docs/kubectl-plugin.md).
```
Just a comment I see this was linked in this format previously so should be ok just making a note in case it needs moved to our number formatting.